hdat/suite.py

```diff
@@ -97,3 +97,79 @@ def _collect_suite_classes(directory):
                     if hdat_suite_class in inspect.getmro(value) and hdat_suite_class != value:
                         suite_classes.append(value)
     return suite_classes
+
+
+class MetricsChecker:
+    '''
+    Helper class for comparing metrics of new results vs golden results in suite.check()
+    '''
+    def __init__(self, old, new):
+        self._old = old
+        self._new = new
+        self._match = True
+        self._msgs = []
+        if old.keys() != new.keys():
+            msg = 'Metric(s) in golden result {} were missing from the new run'
+            missing_metrics = set(old.keys()).difference(set(new.keys()))
+            self._msgs.append(msg.format(missing_metrics))
+            self._match = False
+
+    def _isclose(self, a, b, rel_tol=1e-09, abs_tol=0.0):
+        'Fills in for math.isclose in 3.4'
```

Comment:
I believe according to PEP8, we should use triple strings `'''`.

In particular see:
https://www.python.org/dev/peps/pep-0008/#documentation-strings
https://www.python.org/dev/peps/pep-0257/#one-line-docstrings